mathml/relations/css-styling/first-line-first-letter-pseudo-elements-003-ref.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>MathML and ::first-line/::first-letter pseudo-elements (reference)</title>
+<p>PASS if there is no red.</p>
+<ol>
+  <li>
+    <div class="redfirstline">
+      <div style="display: inline math">Hello,<br/>World!</div>
```

Comment:
Please use tags `<math>` in the reference too, otherwise it's very unlikely they will match (e.g. special math font). More generally, the changes between the test and the ref should be as minimal as possible.